packages/pn-commons/src/components/TitleBox.tsx

```diff
@@ -37,16 +37,34 @@ export default function TitleBox({
   children,
 }: Props) {
   return (
-    <Grid container mt={mtGrid} sx={sx}>
-      {title && <Grid item xs={12} mb={mbTitle}>
-        <Typography variant={variantTitle}>{title}</Typography>
-      </Grid>}
-      {subTitle && <Grid item xs={12} mb={mbSubTitle}>
-        <Typography variant={variantSubTitle} sx={{ fontSize: '18px' }} component={typeof subTitle !== "string" ? "div" : "p"}>
-          {subTitle}
-        </Typography>
-      </Grid>}
-      <Grid item xs={12}>
+    <Grid id="container" aria-orientation="horizontal" tabIndex={0} container mt={mtGrid} sx={sx}>
```

Comment:
Not sure about this id prop. The value `container` is too generic but also what's purpose?

Reply:
correct with different id